openedx_events/__init__.py

```diff
@@ -5,4 +5,4 @@
 more information about the project.
 """
 
-__version__ = "9.5.1"
+__version__ = "9.5.2"
```

Comment:
Given that we are removing events that were public, and might be used by any other developer that imported this code, this should be a breaking change with a major version bump.

Naturally the events are not in use by edx-plaform right now. Where they ever used by that repo?

Reply:
Nope, I think this would be safe to merge w/o considering a major bump since the 'permission' models this event would be based on don't even exist. I can confidently say this has never been used. That said, I don't have strong options on if this bumps a major version or not. Just figured I'd point that out if we'd like to avoid going to v10.

In hindsight, we should not have included these until we were more certain the new permission model would get merged into edx-platform.

Reply:
If this was never really releaseda anywhere and has never been used I'm good with not bumping for major.